compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -2340,7 +2340,7 @@ static LogicalResult setRootConfig(IREE::GPU::TargetAttr target,
     LDBG("Tile and fuse data tiled multi_mma config");
     return success();
   }
-  if (clGPUTestTileAndFuseMatmul) {
+  if (clGPUEarlyTileAndFuseMatmul) {
```

Comment:
I don't understand enough about the whole process here, but just want to ask if this will make matmul tile and fuse to bail for those cases that could actually beat vector distribute? Does tuning needs to take this into consideration?

Reply:
it will right now, but we want to land #19854 or @nirvedhmeshram's version of that PR (idk if he put up a draft for it) and then tile and fuse will always take precedence. The fact that we have so many pipeline is just a transient thing, we're trying to burn them down.

Reply:
Got ya, thanks for the explanation.